app/converters/gbfs_nextbike_vehicle_availabilities.py

```diff
@@ -59,6 +59,8 @@ def _convert_station_status(self, system_id: str, data: dict, path: str) -> dict
             return data
 
         vehicles = self.free_vehicles_cache_per_system.get(system_id, [])
+        if not vehicles:  # _convert_free_vehicles_status has not yet been called for this system_id
```

Comment:
I suppose this will not work, as `get(system_id, [])` will return an empty list, if no vehicles have been cached, so better check against `len(vehicles) == 0`